Packs/CheckPointHEC/Integrations/CheckPointHEC/CheckPointHEC.yml

```diff
@@ -1,6 +1,6 @@
 category: Email
 commonfields:
-  id: CheckPointHEC
+  id: Check Point Harmony Email and Collaboration (HEC)
```

Comment:
Please revert, you cannot change the `id` of an integration, see [build failure](https://github.com/demisto/content/actions/runs/5567162789/jobs/10168829275?pr=28109#step:7:15).
```suggestion
  id: CheckPointHEC
```

Reply:
@kgal-pan does this mean it is not possible to change the integration name?:
```
[2023-07-16 17:56:03,768] [ERROR] Packs/CheckPointHEC/Integrations/CheckPointHEC/CheckPointHEC.yml: [BA101] - The name attribute of CheckPointHEC.yml (currently Check Point Harmony Email and Collaboration (HEC)) should be identical to its `id` attribute (CheckPointHEC)
```

Reply:
Correct. This is because the name and id of the integration are used as an index to be able to search and find the Pack (and also construct the URI to to Pack, e.g https://cortex.marketplace.pan.dev/marketplace/details/CheckPointHEC/. You can change the display name though.

Reply:
Thanks, changed only display name.